OpenPedCan-exprs-shinyapp/server.R

```diff
@@ -0,0 +1,167 @@
+# source functions
+source('R/pubTheme.R')
+source('R/viewDataTable.R')
+source('R/pan_cancer_plot.R')
+source('R/tumor_vs_normal_plot.R')
+
+# Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+## global data read each session
+# expression matrix and histology file
+expr_mat <- readRDS('data/gene-expression-rsem-tpm-collapsed.rds')
+genes <- unique(rownames(expr_mat))
+hist_file <- read.delim('data/histologies.tsv', stringsAsFactors = F)
+
+# mapping files for ENSEMBL identifier, RMTL, EFO and MONDO codes
+ensg_hugo_rmtl_mapping <- read.delim('data/ensg-hugo-rmtl-mapping.tsv') %>%
+  unique()
+efo_mondo_map <- read.delim('data/efo-mondo-map.tsv') %>%
+  unique()
+
```

Comment:
Will we have to point these to the s3 paths instead @komalsrathi - cc @blackdenc and @jonkiky

Reply:
@jharenza I will have to explore on how to do that. My guess is it would need aws authentication in some config file to access the data and should not be difficult but I don't have much time to explore this right now so can we please stick with `data/` for now unless someone else wants to take this on.

Reply:
Alternatively, we can just clone the `OpenPedCan-analysis` repo on the same server where the scripts are going to be hosted, run the download script with each version and point the `shiny-app/data/` to `OpenPedCan-analysis/data`